Assets/Scripts/Augment/AugmentImplementations/StickyDynamite.cs

```diff
@@ -44,10 +44,18 @@ public void SetBarrel()
 
     public void Detonate(PlayerManager sourcePlayer)
     {
-        mesh.enabled = false;
-        explosiveBarrel.enabled = false;
-        smoke.enabled = false;
-        barrelSmoke.ToList().ForEach(smoke => smoke.enabled = false);
+        // Needs extra if-guards in case resetdynamite is called asynchronously by server
```

Comment:
Dunno about the if guards here, might be that the try-catch block is enough as a safeguard for crashes. However, if we end up with a dynamite stick that is in a bad state for some reason... sure, keep it in